### ISSUE TITLE
Consider supporting readthedocs

### ISSUE THREAD

**Issue by benoit74** [openzim/overview#45]
Currently, we use the free Community plan of readthedocs.com

readthedocs is becoming an important cornerstone of our documentation since we've already published libzim, libkiwix, zim-tools and python-scraperlib documentation there, and we plan to add more. I think we could / should even publish all scrapers documentation there rather than using the crappy Github wiki.

The free Community plan is mostly sufficient except that Ads are displayed throughout our documentation, see e.g. https://python-scraperlib.readthedocs.io/en/latest/

For 5$ per month, we could remove these adds and give support to readthedocs.
 
<img width="773" alt="Image" src="https://github.com/user-attachments/assets/cdc5df68-3404-4300-95e6-f8b80eb2a2a3" />

See https://about.readthedocs.com/pricing/#/community

Given the moderate monthly price plus the fact that it is important that we also give back to teams we like, I think that we should move to this plan. 

WDYT?

**Comment by rgaudin** [comment]
I agree. I believe @mgautierfr raised this question in the past when we started using it. I dont recall the discussion. Maybe at the time we were stil testing the platform.

**Comment by kelson42** [comment]
OK on the principle to pay RTD. Segregation of dities between wiki and RTD is an interesting topic...

**Comment by Popolechien** [comment]
LGTM. I created an account, but do I individually sign up for monthly support or do I need to be attached in any manner to the Kiwix docs?

**Comment by benoit74** [comment]
> I created an account, but do I individually sign up for monthly support or do I need to be attached in any manner to the Kiwix docs?

That's a good point, we do not have an organization ... I hope we do not have to pay 5$ per project ... Hopefully you just need to be a maintainer of our projects to remove the adds ... we should probably ask them

**Comment by benoit74** [comment]
I sent them a mail

**Comment by benoit74** [comment]
> An owner of the projects needs to have a gold membership in order to remove ads from a project.
For each $5 a month, you can remove ads from one project ($10, two projects, and so on).
Marking projects ad free is a manual process (https://app.readthedocs.org/accounts/gold/projects/), but it just needs to be done just once per project.

This makes the thing a bit different in term of budget (we currently have 4 projects, but probably more are coming). Given readthedocs limited added value (we could pretty easily do the same with our own infra and a Github workflow, it is just a bit more convenient in Github pages), I'm not convinced anymore. I will open an issue about a proposition to host this documentation on our own infra.

**Comment by kelson42** [comment]
@benoit74 We can not have a custom price wuth a flatrate?

**Comment by benoit74** [comment]
It does not look so.
